kagglesdk/kernels/types/kernels_api_service.py

```diff
@@ -1317,6 +1317,8 @@ class ApiSaveKernelRequest(KaggleObject):
       Which docker image to run with. This must be one of the Kaggle-provided,
       known images. It should look something like:
       gcr.io/kaggle-images/python@sha256:f4b6dd72d4ac48c76fbb02bce0798b80b284102886ad37e6041e9ab721dc8873
+    kernel_version_type (KernelVersionType)
+      Which kernel version type to use.
```

Comment:
See comment below about naming.  I think this should be called `kernel_execution_type`.  Also, might be nice to document that the default is `SAVE_AND_RUN_ALL`.